handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -464,14 +464,8 @@ private SSLEngineResult readPendingBytesFromBIO(
     }
 
     @Override
-    public synchronized SSLEngineResult wrap(
+    public SSLEngineResult wrap(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "wrap" is 41 which is greater than 25 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)